.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name:  CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - uses: borales/actions-yarn@v2.0.0
+      with:
+        cmd: install --frozen-lockfile
+    - uses: borales/actions-yarn@v2.0.0
+      with:
+        cmd: lint
+    - uses: borales/actions-yarn@v2.0.0
+      with:
+        cmd: test
+    - uses: borales/actions-yarn@v2.0.0
+      with:
+        cmd: build
```

Comment:
can we add a script that will automatically deploy storybook to GH pages? and also publishes an alpha release? just like we did in travis ci

Reply:
Yeah for sure we can do that.